src/taco-cli/test/build.ts

```diff
@@ -103,6 +103,8 @@ describe("taco build", function (): void {
         kitHelper.kitPackagePromise = null;
         testHttpServer.close();
         rimraf(tacoHome, function (err: Error): void { done(); }); // ignore errors
+        // Reset mocked out resources
+        process.env["TACO_UNIT_TEST"] = false;
```

Comment:
What I meant was that TACO_HOME and TACO_UNIT_TEST used to be set and reset at the same time. Now that you modified the behavior for TACO_UNIT_TEST, do we have to do the same for TACO_HOME?